extension/content/utils/fetchWithTimeout.js

```diff
@@ -0,0 +1,27 @@
+export default function fetchWithTimeout(url, options = {}, duration = 5000) {
```

Comment:
the need for this appears to be to address the HTTP/2 TCP issue we discovered. if we can find a workaround for that we likely don't need (or want) a timeout on the fetch.

Reply:
on second thought i think we should keep this

Reply:
Yeah. I think it would even be good if timeout was the default way we made requests everywhere.

Reply:
yeah, makes sense